country.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'country_module'
+class Country
+
+  include Check_country
+
+  def initialize(country_name ,population, gross_domestic_production, states, army_strength, state_of_country)
```

Comment:
after each parameter add comma immediately, and then add space.
`country_name ,population,`

to

`country_name, population,`